### ISSUE TITLE
Tab context menu entries should have tooltips

### ISSUE THREAD

**Issue by vadimkantorov** [microsoft/terminal#13243]
This is sometimes needed as a last resort to find some info printed to the screen but unsaved anywhere else. Currently for doing this I had to do manual selection with mouse and then wait for ~1min for it to select all of scrollable viewport (hoping that selection doesn't break since it's quite jumpy), then copy to Notepad, then search.

So either allowing to copy the scrollable viewport into buffer and/or searching it directly can be useful.

**Comment by zadjii-msft** [comment]
You mean something like:
* [`selectAll`](https://docs.microsoft.com/en-us/windows/terminal/customize-settings/actions#select-all-preview), [`copy`](https://docs.microsoft.com/en-us/windows/terminal/customize-settings/actions#copy)
* Alternatively, [`exportBuffer`](https://docs.microsoft.com/en-us/windows/terminal/customize-settings/actions#buffer-exporting)
* Or possibly, [`find`](https://docs.microsoft.com/en-us/windows/terminal/customize-settings/actions#find)

/?

**Comment by vadimkantorov** [comment]
Yes, but making them more discoverable would be good. I've tried right-clicking on the tab, there is no menu.

Only after you posted this, I found a context menu on the tab. Export Text proposed `vadimkantorov@dell_de_vadim: /mnt/c/Users/vadim` which is illegal file name.

Export Text should have a tooltip that explains what it does. does it export all scrollable viewport? or just current viewport? will it copy to clipboard or to text file?

Ctrl+Shift+P search bar does not find anything by "export" or "select".

Ctrl+Shift+P -> find does the right thing but it's hard to know it exists. Command Palette fast typing also makes for slow rendering, it reacts slower than I type and it's strange. It should not try to re-render the drop-down too fast, better to skip some intermediate searches than re-render and flicker (also it seems a bit slow at rendering, XAML bites again I suppose)

**Comment by zadjii-msft** [comment]
> right-clicking on the tab (it seems to be doing PasteText in my case), there is no menu

That's tracked in #3337 

> Export Text should have a tooltip that explains what it does

Fair - none of those context menu entries have tooltips. Repurposing this issue for that.

> Ctrl+Shift+P search bar does not find anything by "export" or "select".

What version are you on? `selectAll` was added in 1.14, but `exportBuffer` was added to the default actions in https://github.com/microsoft/terminal/pull/12097, which should appear as "Export Buffer" in english.

**Comment by vertigo220** [comment]
This issue deals with tooltips explaining a menu item's functionality, whereas issue #13588 dealt with actual text next to the context menu entries to show their associated hotkey, so I'm not sure if it's really a duplicate of this, but I wanted to clarify here the need for hotkey info, preferably as instantly viewable text vs a tooltip that requires hovering to see.

**Comment by zadjii-msft** [comment]
That's what I get for trying to triage on vacation 😅
